pillar/heroku/mitxonline.sls

```diff
@@ -17,6 +17,7 @@
       'MAILGUN_FROM_EMAIL': 'MITx Online <no-reply@mitxonline-rc-mail.mitxonline.mit.edu>',
       'MAILGUN_SENDER_DOMAIN': 'mitxonline-rc-mail.mitxonline.mit.edu',
       'MITOL_PAYMENT_GATEWAY_CYBERSOURCE_SECURE_ACCEPTANCE_URL': 'https://testsecureacceptance.cybersource.com/pay',
+      'MITOL_PAYMENT_GATEWAY_CYBERSOURCE_REST_API_ENVIRONMENT': 'apitest.cybersource.com',
```

Comment:
This URL like string without protocol is intentional, needed as is by the Python client we use for CyberSource APIs.